rts/Rml/SolLua/README.md

```diff
@@ -0,0 +1,65 @@
+RmlSolLua
```

Comment:
This readme has to be heavily updated to describe why this library is vendored in the first place. The last time I looked at it it was used as a base of code, but very heavily modified to suit our needs, effectively making it ipossible to include as submodule. This readme should describe it.